providers/dns/brandit/brandit.toml

```diff
@@ -1,5 +1,10 @@
-Name = "Brandit"
-Description = ''''''
+Name = "Brandit (deprecated)"
+Description = '''
+Brandit has been acquired by Abion.
+Abion has a different API.
```

Comment:
Does this mean the API is already defunct? If so, we could prune the provider similar to what I suggested in #2324 (i.e. immediately return an `errors.New("brandit: provider defunct, migrate to albion")` or similar)

Reply:
we don't know, maybe their API still works, so I prefer to not drop it for now.

Reply:
I see. In this case, LGTM!